client/src/components/BuildingDetails.js

```diff
@@ -16,52 +16,65 @@ export default function BuildingDetails(props) {
       style={{ position: 'absolute' }}
     >
       <Title className="building-details-title" level={3}>
-        <Trans>Building Info</Trans>
+        <Row>
+          <Col lg={1} span={0}>
+            <Icon type="environment" />
+          </Col>
+          <Col lg={{ span: 22, offset: 1 }} span={24}>
+            <Trans>Building Info</Trans>
+          </Col>
+        </Row>
       </Title>
       {details ? (
         <div>
-          <p>
-            <strong>
+          <Row>
+            <Col lg={{ span: 13, offset: 2 }} span={13}>
               <Trans>Address</Trans>:
-            </strong>
-            {details.address}
-          </p>
-          <p>
-            <strong>
+            </Col>
+            <Col lg={9} span={11}>
+              {details.address}
```

Comment:
The building address is composed of the street name and the street number
The `postcode` actually contains the street number information.

Reply:
So `{details.address} {details.postcode}`?

Reply:
Almost; it's more like this:
```
{details.address} no. {details.post_code}  // for English
{details.address} nr. {details.post_code}  // for Romanian
```

I know it's confusing to use the `post_code` for this but we'll get around and change the model & API to eliminate the confusion.

Reply:
any formatting for `postcode`? trim leading zeros?

Reply:
Yes, I think you can do that for good measure 🤔